expression/aggregation/agg_to_pb_test.go

```diff
@@ -52,17 +42,16 @@ type testEvaluatorSuite struct {
 	ctx sessionctx.Context
 }
 
-func (s *testEvaluatorSuite) SetUpSuite(c *C) {
+func createtestEvaluatorSuite(t *testing.T) (s *testEvaluatorSuite) {
+	s = new(testEvaluatorSuite)
 	s.Parser = parser.New()
 	s.ctx = mock.NewContext()
+	return
```

Comment:
This suite is redundant also - you can just inline the setup lines.

Reply:
Fixed - 73c9037